src/main/kotlin/org/rust/cargo/project/model/impl/CargoProjectImpl.kt

```diff
@@ -339,7 +339,9 @@ data class CargoProjectImpl(
     }
 
     override val presentableName: String
-        get() = workingDirectory.fileName.toString()
+        get() = projectService.allPackages.singleOrNull {
```

Comment:
Also, you don't need to iterate all available packages in project. Only packages in `workspace` of currect project with `origin == STDLIB`

Reply:
I guess you meant `origin == WORKSPACE`, which I implemented now

Reply:
Oops. Yep, sorry :sweat_smile: